README.md

```diff
@@ -12,7 +12,7 @@ A Client library for OAuth2 which uses JWT for the generation of access tokens.
               .subject("myapp ")
               .build();
 
-  TokenSource tokenSource = config.tokenSource();
+  TokenSource tokenSource = TokenSources.reusableTokenSource(new OkHttpTokenSource(config));
```

Comment:
BearerAuthenticationInterceptor - why not Oauth2AuthenticationInterceptor or ClouwayJwtBearerAuthenticationInterceptor or JwtBearerAuthenticationInterceptor

sometinig more specific in the name, because  BearerAuthenticationInterceptor uses JwtConfig - so it is strongly JWT token related bearer authenticated.

just wondering about the names